Header header

uint32 all_clovers
uint32 connected_clovers
uint32 failed_clovers
uint32 armed_clovers

uint32[] all_clovers_ids
uint32[] connected_ids
uint32[] failed_ids
uint32[] armed_ids
uint32[] with_led_ids
uint32[] with_camera_ids